src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -161,12 +144,12 @@ const mapStateToProps = state => {
 
   return {
     appointments: state.health?.appointments?.data,
+    shouldFetchMessages: state.user.profile?.services?.includes(
```

Comment:
Could you use the `createIsServiceAvailableSelector` for this? Or at least the existing `selectAvailableServices` selector?

Reply:
Done!

Reply:
And yes.. good point. Updated the logic to also display `message` 👍